app/assets/stylesheets/globals/fonts.css.scss

```diff
@@ -1,3 +1,11 @@
 // Whitney (sans serif):
-//@import url("http://cloud.typography.com/7419092/765888/css/fonts.css");
-// TODO?
\ No newline at end of file
+@import url("//cloud.typography.com/7419092/6220752/css/fonts.css");
+
+$font-family: "Whitney SSm A", "Whitney SSm B", sans-serif;
```

Comment:
Does this get used anywhere other than defaults.css.scss? If not, maybe just do the actual font-family declaration here? (Just worry about someone tweaking default.css at some point, but not cleaning this up, and then someone else tries to make a tweak here... all rather hypothetical.)